packages/orbit-components/src/Wizard/index.js

```diff
@@ -0,0 +1,154 @@
+// @flow
+import * as React from "react";
+import { css } from "styled-components";
+
+import WizardStep from "./WizardStep";
+import { WizardStepContext } from "./WizardContext";
+import Button from "../Button";
+import ButtonLink from "../ButtonLink";
+import Stack from "../Stack";
+import ChevronDown from "../icons/ChevronDown";
+import Portal from "../Portal";
+import Modal from "../Modal";
+import { CardSection } from "../Card";
+import useMediaQuery from "../hooks/useMediaQuery";
+import useTheme from "../hooks/useTheme";
+
+import type { Props } from ".";
+
+const Wizard = ({ dataTest, id, completedSteps, activeStep, children, onChangeStep }: Props) => {
+  const { isTablet } = useMediaQuery();
+  const theme = useTheme();
+  const [open, setOpen] = React.useState(false);
+  const toggle = React.useRef<React.ElementRef<typeof ButtonLink> | null>(null);
+
+  const childrenArray = React.Children.toArray(children);
+  const stepStatuses = childrenArray.map((step, index) => {
+    let status;
+    if (index < completedSteps) {
+      status = "completed";
+    } else if (index === completedSteps) {
+      status = "available";
+    } else {
+      status = "disabled";
+    }
+    return status;
```

Comment:
probably this will look better:

```
if (index < completedSteps) return "completed"
if (index === completedSteps) return "available";
return "disabled";

Reply:
Yep 👍